### ISSUE TITLE
`@property` should map to an LSP Property kind (or Field)

### ISSUE THREAD

**Issue by rwols** [muffinmad/anakin-language-server#15]
```python
class Foo:

    @property
    def bar(self):
        return 42


foo = Foo()
foo.b|
```
This will report `bar` as a function, but it should be a property (or field).

**Comment by muffinmad** [comment]
The simple and brittle solution that comes to mind is to look if line before completion candidate definition is `@property`.

But let's expand Jedi's types range by little if we can! ;)

@davidhalter Can you please tell whether it is possible to make this test pass in Jedi:

```python
def test_completion_property(Script):
    script = Script(dedent('''\
        class Foo:

            @property
            def bar(self):
                return 42


        foo = Foo()
        foo.b
        '''))
    c, = script.complete(9, 5)
    assert c.type == 'property'
```

Thanks!

**Comment by davidhalter** [comment]
Going to include that in 0.18.0 probably. But note that this is not going to be perfect and more like a heuristic (which is probably good enough).

**Comment by davidhalter** [comment]
This will be fixed in 0.18.0. IMO it was never really a bug, but it's definitely nicer this way.

**Comment by muffinmad** [comment]
@davidhalter Awesome! Thanks!

@rwols You can install latest Jedi's master, `anakinls` v1.18.1 already supports Jedi's `property` completion type.
